styleguide/source/_patterns/03-templates/04-topic/topic.twig

```diff
@@ -5,13 +5,9 @@
 
 <main class="topic container">
 
-  <div class="grid grid-margin">
-    <div class="topic_top grid-region">
-      {% include 'atoms-page-title' %}
-    </div>
-  </div>
 
-  <div class="topic_main grid-margin">
+
+  <div class="topic_main grid-margin css-grid">
```

Comment:
If this is going to require new styles you'll need to either open a corresponding PR on D8 or create a ticket for someone else to do that. Please link either the PR or the ticket before this one is merged.